packages/vertexai/src/requests/request.ts

```diff
@@ -42,8 +42,7 @@ export class RequestUrl {
     public requestOptions?: RequestOptions
   ) {}
   toString(): string {
-    // TODO: allow user-set option if that feature becomes available
-    const apiVersion = DEFAULT_API_VERSION;
+    const apiVersion = this.requestOptions?.apiVersion || DEFAULT_API_VERSION;
```

Comment:
I think if we are going to go with the enum, we had better go into constants.ts and assign the new enum to DEFAULT_API_VERSION so we don't have 2 sources of truth for the "v1beta" string (is v1beta the default still or is v1?). Or maybe just get rid of the DEFAULT_API_VERSION in constants.ts since it's not a constant anymore and do the DEFAULT_API_VERSION assignment here in this file.

Reply:
I _think_ I prefer the plan of getting rid of the `DEFAULT_API_VERSION` in constants.ts and setting it here.

In this PR the default is still `v1beta` but the plan is to move to `v1` as the default alongside the release of Imagen (requiring devs to switch to `v1beta` to use it since it'll be Public Preview, not GA).